src/plugins/intel_gna/src/preprocessing.cpp

```diff
@@ -3,11 +3,14 @@
 //
 
 #include "preprocessing.hpp"
+#ifdef HAVE_AVX2
+#    include <immintrin.h>
```

Comment:
why do you need this header here? 
I don't that intrinsics are used here

Reply:
You're right, this was a leftover after a refactor. I've removed it now.